packages/odf/components/s3-iam/iam-user-details/iam-accesskeys-details/AccessKeysDetails.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import { AccessKeyMetadata } from '@aws-sdk/client-iam';
+import { IAMUserDetails } from '@odf/core/types';
+import { useCustomTranslation } from '@odf/shared';
+import { SectionHeading } from '@odf/shared/heading/page-heading';
+import { MAX_ACCESS_KEYS } from '@odf/shared/iam';
+import { useModal } from '@openshift-console/dynamic-plugin-sdk';
+import { ModalComponent } from '@openshift-console/dynamic-plugin-sdk/lib/app/modal-support/ModalProvider';
+import {
+  Alert,
+  AlertVariant,
+  Button,
+  Grid,
+  Spinner,
+} from '@patternfly/react-core';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import { GenerateAnotherAccessKeyModal } from '../../../../modals/s3-iam/GenerateAnotherAccessKeyModal';
+import './AccessKeysDetails.scss';
+import AccessKeyCard from './AccessKeyCard';
+
+type IAMAccesskeysDetailsProps = {
+  obj: IAMUserDetails;
+};
+
+/**
+ * Displays AccessKeysDetails in a Card.
+ * Create Another Accesskey if only one Accesskey exists
+ * @param userName @param iamAccessKeys @param tags @param refetchAll @param noobaaIAM
+ */
+export const IAMAccessKeysDetails: React.FC<IAMAccesskeysDetailsProps> = ({
+  obj,
+}) => {
+  const { t } = useCustomTranslation();
+  const launchModal = useModal();
+
+  const { userName, userDetails, iamAccessKeys, refetch, noobaaIAM } = obj;
+  const isLoading = false;
+
+  const launchModalOnClick = (modalComponent: ModalComponent) => () => {
+    launchModal(modalComponent, {
+      isOpen: true,
+      userName,
+      refetch,
+      noobaaIAM,
+    });
+  };
+
+  return (
+    <div className="odf-m-pane__body">
+      <div className="row pf-v5-u-mt-md">
```

Comment:
`row` is not a class...
```suggestion
      <div className="pf-v5-u-mt-md">
```